src/chrome/content/rules/trakt.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+	Mixed content on secure:
+
+		- Images, from:
+
+			- i0.wp.com (from Gravatar)
+
+-->
+<ruleset name="trakt" platform="mixedcontent">
+
+	<target host="trakt.tv" />
+	<target host="*.trakt.tv" />
+
+	<!-- For securecookie rule below (host used for movie etc. images) -->
+	<target host="*.trakt.us" />
+
+	<test url="http://www.trakt.tv/" />
```

Comment:
I'm recommending that for rules like this, you replace

`<target host="*.trakt.tv">`

```
<target host="www.trakt.tv" />
<target host="support.trakt.tv" />
```

That way you can leave out the test URLs, since each non-wildcard target tag creates an implicit test URL.

Also, that way you can greatly simplify the rewrite rule, so it becomes:

```
<rule from="^http:"
  to="https:" />
```

Thanks,
Jacob